### PR TITLE
Cert update

### PR DESCRIPTION
**Pull request by potatoqualitee**

### CODE REVIEW

File: functions/Import-DbaCertificate.ps1 [hunk 1]
```diff
@@ -41,62 +43,78 @@ Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
...
+				}
+			}
+		}
+		
+		try {
+			Write-Message -Level Verbose -Message "Connecting to $SqlInstance"
+			$server = Connect-SqlServer -SqlServer $SqlInstance -SqlCredential $sqlcredential
+		}
+		catch {
+			Stop-Function -Message "Failed to connect to: $SqlInstance" -Continue -Target $SqlInstance -InnerErrorRecord $_
```
Comment (FriedrichWeinmann) [inline functions/Import-DbaCertificate.ps1]: Wrong usage. Don't use the `-Continue` parameter outside a loop.
Use `return` on next line instead

File: functions/Import-DbaCertificate.ps1 [hunk 2]
```diff
@@ -41,62 +43,78 @@ Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
...
-    END {
-		$server.ConnectionContext.Disconnect()
-
+	process {
+		if (Test-FunctionInterrupt) { return }
+		
+		foreach ($fullname in $path) {
+			
+			if (![dbavalidate]::IsLocalhost($SqlInstance) -and !$fullname.StartsWith('\')) {
+				Stop-Function -Message "Path must be a UNC share when SQLInstance is not local." -Continue
```
Comment (FriedrichWeinmann) [inline functions/Import-DbaCertificate.ps1]: Add target and tell the user what path it is

File: functions/Import-DbaCertificate.ps1 [hunk 3]
```diff
@@ -41,62 +43,78 @@ Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
...
+			}
+			
+			if (!(Test-SqlPath -SqlInstance $server -Path $fullname)) {
+				Stop-Function -Message "$SqlInstance cannot access $fullname" -Continue
```
Comment (FriedrichWeinmann) [inline functions/Import-DbaCertificate.ps1]: Add target (the fullname)

**Comment by potatoqualitee** [comment]
merci beaucoup . i am actually going to merge these then rework them with other new commands. there's a failing test on dev w/o it.

**Pull request merged**